tests/backend/test_fock_measurement.py

```diff
@@ -123,7 +123,7 @@ def test_fock_measurements(self, setup_backend, cutoff, batch_size, pure, tol):
             ref_result = n[meas_modes]
 
             if batch_size is not None:
-                ref_result = tuple(np.array([i] * batch_size) for i in ref_result)
+                ref_result = np.array([[i] * 2 for i in ref_result]).T.reshape((batch_size, 1, ref_result.shape[0]))
```

Comment:
could you help me out why ` * 2` is included?

Reply:
Oh, it should be `batch_size` instead of `2`. It just happens that `batch_size = 2` in these tests, so I think I wrote it in like that instead. The transposition is needed since the batches should be arranged opposite of what the list comprehension gives, and the reshaping since the results should be nested (i.e. adding the `1` inside the shape `(batch_size, 1, ref_result.shape[0])`.

Reply:
sounds reasonable, thanks!